src/ape/cli/choices.py

```diff
@@ -126,13 +126,30 @@ def __init__(
     def convert(
         self, value: Any, param: Optional[Parameter], ctx: Optional[Context]
     ) -> Optional[AccountAPI]:
+        if isinstance(value, str) and value.startswith("TEST::"):
+            try:
+                account_idx = int(value.replace("TEST::", ""))
```

Comment:
1 bit of feedback (that i will address):  try to limit what you put in the try / catch to _only_ include the part that will error.

for example, this should be refactored to:

```python
if isinstance(value, str) and value.startswith("TEST::"):
    value_str = value.replace("TEST::", "")
    try:
        account_idx = int(value)
        ....
```

however, even better is avoiding the try catch altogether and just using the `isnumeric()` method.
try / catch is last resort for a situation like this